README.md

```diff
@@ -1,23 +1,21 @@
 <h1 align="center"> <a href="https://www.jsdelivr.com/globalping"><img width="28" alt="Globalping icon" src="https://user-images.githubusercontent.com/1834071/216975126-01529980-a87e-478c-8ab3-bf7d927a1986.png"></a> Globalping CLI </h1>
 
-<p align="center">Access a global network of probes without leaving your console. Powered by the Globalping community!</p>
+<p align="center">Run measurement commands from a global network of probes right from your console with the official command-line interface for <a href="https://github.com/jsdelivr/globalping">Globalping</a>. Powered by the Globalping community!</p>
```

Comment:
Not sure about this. I wanted to avoid "measurements" because it sounds like its limited to benchmarking. Instead, having access means you are more flexible to do what you need.

Reply:
I see what you mean! I still think we need to provide some more context and info so that we don't lose users who are not familiar with Globalping yet. 

How about something like (text from https://www.jsdelivr.com/globalping/cli):

Access a global network of probes without leaving your console. Run network commands, write scripts, automate uptime monitoring, do latency checks, or make sure your anycast network continues to perform up to your standards, globally! 
Powered by the Globalping community!

Reply:
Sure, just make the text unique for better SEO

Reply:
Updated the text accordingly!